api/src/pcapi/models/feature.py

```diff
@@ -127,6 +127,7 @@ class FeatureToggle(enum.Enum):
     WIP_OPENING_HOURS = "Activer les horaires d'ouverture sur la page partenaire"
     WIP_ENABLE_ADAGE_VISUALIZATION = "Activer la nouvelle manière de visualiser les offres dans la recherche d'ADAGE."
     WIP_ACCESLIBRE = "Activer l'affichage des données Accès Libre sur les pages partenaires"
+    WIP_ENABLE_TITLIVE_API_FOR_BOOKS = "Activer l'utilisation de l'API Titelive pour les synchronisations produits pour les livres(désactive l'utilisation de l'utilisation de FTP pour la synchro)"
```

Comment:
Je ne doute pas un instance que "titlive" sans "e" existe, mais on travaille plutôt avec TiteLive avec un "e". ;)

```suggestion
    WIP_ENABLE_TITELIVE_API_FOR_BOOKS = "Activer l'utilisation de l'API Titelive pour les synchronisations produits pour les livres(désactive l'utilisation de l'utilisation de FTP pour la synchro)"
```